### DISCUSSION TITLE
Overriding CSS

### DISCUSSION THREAD

**Discussion by Karthiksaravanan11** [squidfunk/mkdocs-material#5764] [Q&A]
Can we override main.css?

**Comment by squidfunk** [comment]
Sure, [it's all documented](https://squidfunk.github.io/mkdocs-material/customization/#additional-css). Just add a CSS file and override the selectors targeting elements that you want to change.

**Reply by Karthiksaravanan11** [reply]
Should I name the css file as main.css or any other naming convention can be given?
